test/functional/p2p_orphan_handling.py

```diff
@@ -566,6 +569,47 @@ def test_orphan_txid_inv(self):
         assert tx_child["txid"] in node_mempool
         assert_equal(node.getmempoolentry(tx_child["txid"])["wtxid"], tx_child["wtxid"])
 
+    @cleanup
+    def test_max_orphan_amount(self):
+        self.log.info("Check that we never exceed our storage limits for orphans")
+
+        self.generate(self.wallet, 160)
```

Comment:
Generating 160 blocks might be overkill. The following generates one and moves `node = self.nodes[0]` to just under the opening log (to match other functions).

```diff
diff --git a/test/functional/p2p_orphan_handling.py b/test/functional/p2p_orphan_handling.py
index 0a88b4c9587..5276814a087 100755
--- a/test/functional/p2p_orphan_handling.py
+++ b/test/functional/p2p_orphan_handling.py
@@ -572,9 +572,9 @@ class OrphanHandlingTest(BitcoinTestFramework):
     @cleanup
     def test_max_orphan_amount(self):
         self.log.info("Check that we never exceed our storage limits for orphans")
-
-        self.generate(self.wallet, 160)
         node = self.nodes[0]
+
+        self.generate(self.wallet, 1)
         peer_1 = node.add_p2p_connection(P2PInterface())
 
         self.log.info("Check that orphanage is empty on start of test")
```

Reply:
Thanks! Updated in [5c299ec](https://github.com/bitcoin/bitcoin/pull/31040/commits/5c299ecafe6f336cffa145d28036b04b87e26712)